superdesk/io/webhooks/__init__.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013 - 2017 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+from superdesk.io.commands import update_ingest
+from flask import request, abort
+import os
+import superdesk
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class FeedinServiceWebhookAuth(object):
+
+    def authenticate(self):
+        abort(403, description='You are not authorized to access this resource')
+
+    def authorized(self, allowed_roles, resource, method):
+        try:
+            # provider's id may be used in the future
+            provider_name = request.args['provider_name']
```

Comment:
@petrjasek actually forget my last message, I've misread the diff and the lookup is OK.
the provider_name is needed here for checking the environment variable, and the page is not authorised if nothing is set in environment, so that's kind of needed here.

It may still be changed later when we have some client UI, because at this time the environment variable should not be used anymore.